WPFHospitalEditor/SearchResultDialog.xaml.cs

```diff
@@ -375,5 +407,22 @@ private string[] MapObjectToContentRows()
             }
             return contentRows;
         }
+
+        private string[] AppointmentToContentRows()
+        {
+            string[] contentRows = new string[HospitalMap.appointemntSearchResult.Count()];
+            for (int i = 0; i < HospitalMap.appointemntSearchResult.Count(); i++)
+            {
+                MapObject mo = mapObjectController.findMapObjectById(HospitalMap.appointemntSearchResult.ElementAt(i).RoomId);
+                string doctor = HospitalMap.appointemntSearchResult.ElementAt(i).Doctor.Person.Name + " " + HospitalMap.appointemntSearchResult.ElementAt(i).Doctor.Person.Surname;
+                string timeInterval = HospitalMap.appointemntSearchResult.ElementAt(i).TimeInterval.Start.ToString() + "-" + HospitalMap.appointemntSearchResult.ElementAt(i).TimeInterval.End.ToString();
```

Comment:
Maybe to use some local var for HospitalMap.appointemntSearchResult.ElementAt(i)

Reply:
fixed